test_lib/cql_types.py

```diff
@@ -0,0 +1,200 @@
+import random
+
+
+__ALL__ = ['CSQLColumnType', 'ALL_COLUMN_TYPES', 'NOT_EMBEDDABLE_COLUMN_TYPES', 'COLLECTION_COLUMN_TYPES']
+
+
+ALL_COLUMN_TYPES = ('ascii', 'blob', 'counter', 'decimal', 'duration', 'int', 'map', 'smallint', 'time',
+                    'timeuuid', 'uuid', 'varint', 'bigint', 'boolean', 'date', 'double', 'float', 'inet', 'list', 'set',
+                    'text', 'timestamp', 'tinyint', 'varchar')
+NOT_EMBEDDABLE_COLUMN_TYPES = ['duration', 'counter']
+COLLECTION_COLUMN_TYPES = ['set', 'list', 'map']
+
+
+class CSQLColumnType:
+    maintype = None
```

Comment:
you are mixing class attributes and instance attributes (have same name)
it is dangerous.